app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/pocket/PocketStoriesViewHolder.kt

```diff
@@ -76,21 +87,34 @@ fun PocketStories(
         }
     }
 
-    ExpandableCard(
-        Modifier
-            .fillMaxWidth()
-            .padding(top = 40.dp)
-    ) {
-        PocketRecommendations {
-            Column {
-                PocketStories(stories ?: emptyList(), client)
+    FirefoxTheme {
```

Comment:
I think we should move this to where `setContent` is. If I think about that future `HomeScreen() {}` composable function that wraps all our home screen feature. We wouldn't necessarily have to specify `FirefoxTheme` at the feature composable level (if that makes sense).

Reply:
Agree. Thanks!